yarn.lock

```diff
@@ -7878,7 +7878,7 @@ mime@1.6.0:
   resolved "https://registry.yarnpkg.com/mime/-/mime-1.6.0.tgz#32cd9e5c64553bd58d19a568af452acff04981b1"
   integrity sha512-x0Vn8spI+wuJ1O6S7gnbaQg8Pxh4NNHb7KSINmEWKiPE4RKOplvijn+NkmYmmRgP68mc70j2EbeTFRsrswaQeg==
 
-mime@^2.4.4, mime@^2.4.6:
+mime@^2.4.6:
   version "2.6.0"
```

Comment:
This change intentional??

Reply:
was missing from the previous installation. I think Bucky merged it in. When I tried to pull and merge master it said I was up to date 🤷🏽